### PR TITLE
Fix formatting + mention `composer dump-autoload`

### PR DESCRIPTION
**Pull request by guilliamxavier**
This is a follow-up to #557 (and commit de22a60ab53cd6edc62fdb121788fd469afff224).

### CODE REVIEW

File: Resources/doc/commands/generate_bundle.rst [hunk 1]
```diff
@@ -32,17 +32,26 @@ forget to pass all needed options:
...
+            "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
         },
-        "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
-    },
+
+    ), then execute the following command to regenerate the autoload files:
+
+    .. code-block:: bash
+
+        $ composer dump-autoload
```
Comment (fabpot) [inline Resources/doc/commands/generate_bundle.rst]: This should not be needed in dev, so I would remove this part.

Comment (guilliamxavier) [inline Resources/doc/commands/generate_bundle.rst]: @fabpot I don't know if it should, but it seems needed even in dev (that or a `composer install`, which includes autoloader dump), see for example https://pastebin.com/CF0DEQbt and https://pastebin.com/hy04dQpX
(by the way, in prod we would probably add `-o`)

Comment (fabpot) [inline Resources/doc/commands/generate_bundle.rst]: hmmm, you're right

**Comment by fabpot** [comment]
Thank you @guilliamxavier.

**Pull request merged**